packages/ui-bindings/Unit/getAllUnits.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * A method to get all Units
+ *
+ * @package: REA app
+ * @author:  Steven Fontaine <fontainesw@msoe.edu>
+ * @since:   2017-12-04
+ */
+
+import { connect } from 'react-redux'
+import { gql, graphql, compose } from 'react-apollo'
+
+import { AppState } from '@vflows/store/types'
+import { getActiveLoginToken } from '@vflows/store/selectors/auth'
+
+import { unitInterface } from "./unit";
+
+const query = gql`
+query($token: String) {
+  viewer(token: $token) {
+    allUnits{
+      ...unitInterface
+    }
+  }
+}
+${unitInterface}
+`
+
+export default compose(
+  // bind input data from the store
+  connect((state: AppState) => ({
+    variables: {
+      token: getActiveLoginToken(state),
+    },
+  })),
+
+  graphql(query, {
```

Comment:
Very nice formatting in this compose. :+1:

Reply:
*very quiet I told you so*